### ISSUE TITLE
Form Enhancement - Create barcodes and containers

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#6620]
**Problem to solve**
Right now, the workflow from barcode testing, reserving and creating containers is hard to remember and very obscure -- exacerbated by the fact that collections typically dont have to do this often but it can hamper data entry and management. So with a few edits to our current forms we can make this a LOT easier.

**Proposed Solution**
Currently the Tools Directory has a link to [Barcode: Series Reserve](https://arctos.database.museum/info/barcodeseries.cfm)

We need to update the help text at the top and rename it to [Barcode: Series Reserve and Create Containers](https://arctos.database.museum/info/barcodeseries.cfm)

Current form:
![Firefox_Screenshot_2023-08-09T21-52-51 198Z](https://github.com/ArctosDB/arctos/assets/2523089/3abaf17d-5908-451c-aa8b-bdbbab3eecd0)

Proposed edits are[ embedded in this form](https://docs.google.com/drawings/d/1n-Fl8lDsh-6UreipKKe5C1Fv6QBvT0RRSVDXd7L1U3k/edit?usp=sharing): Please comment or edit.
In a nutshell, it explicit states the steps from barcode testing and reservation to container creation with links to the appropriate forms.

**Comment by Jegelewicz** [comment]
But that form doesn't create containers?

**Comment by Jegelewicz** [comment]
Also - what here needs updating? https://handbook.arctosdb.org/how_to/How-to-Start-Object-Tracking.html

**Comment by mkoo** [comment]
> But that form doesn't create containers?

Ah, yes but it needs a link to the next step... so this is easy to fix by adding clear steps and a link to the needed forms

**Comment by mkoo** [comment]
> Also - what here needs updating? https://handbook.arctosdb.org/how_to/How-to-Start-Object-Tracking.html

we also have no link to this on the form -- we may want to introduce a visually consistent button to relevant handbook links

**Comment by dustymc** [comment]
>visually consistent button

[Manage Arctos Directory](https://arctos.database.museum/Admin/manage_directory.cfm) then pagehelp

<img width="1292" alt="Screenshot 2023-08-09 at 21 46 36" src="https://github.com/ArctosDB/arctos/assets/5720791/c5a462dc-9727-4f55-8429-b9f5182f7fa0">

et voilà!

<img width="481" alt="Screenshot 2023-08-09 at 21 45 10" src="https://github.com/ArctosDB/arctos/assets/5720791/9481129c-d850-4afb-9336-493469050d02">

**Comment by mkoo** [comment]
This requires an inperson discussion. reorganizing into Discussion milestone and assigning to me so we wont forget this! (maybe a UI topic too?)

**Comment by mkoo** [comment]
done!
